lib/spack/spack/cmd/bootstrap.py

```diff
@@ -34,6 +34,9 @@ def setup_parser(subparser):
     cache_group.add_argument(
         '--no-cache', action='store_false', dest='use_cache', default=True,
         help="do not check for pre-built Spack packages in mirrors")
+    cache_group.add_argument(
+        '--cache-only', action='store_true', dest='cache_only', default=False,
```

Comment:
Just a comment, not a request for any change. Using `store_true` as an action already sets the default to `False`. Vice-versa is valid for `store_false`. See [here](https://docs.python.org/2/library/argparse.html#action) for reference. Also setting `dest=cache_only` should be unnecessary, in the sense that coincides with the default given the argument name.